nmtrust

```diff
@@ -78,7 +111,7 @@ no_network() {
     exit 4
 }
 
-while getopts ":e:t:qh" opt; do
+while getopts ":e:t:q:vh" opt; do
```

Comment:
The `:` after the `q` looks like an error. `-q` does not take any arguments.

Reply:
ok. It's fixed